pysr/logger_specs.py

```diff
@@ -74,3 +79,17 @@ def write_hparams(self, logger: AnyValue, hparams: dict[str, Any]) -> None:
                 ],
             ),
         )
+
+    def close(self, logger: AnyValue) -> None:
+        base_logger = jl.SymbolicRegression.get_logger(logger)
+        close_logger = jl.seval(
+            """
+            function close_logger(lg::TensorBoardLogger.TBLogger)
+                # close open streams
+                for k=keys(lg.all_files)
+                    close(lg.all_files[k])
```

Comment:
Is this part needed? I would assume closing the logger itself would handle closing these files. Do the docs say otherwise? Or maybe you’ve seen it’s needed?

Reply:
Oh wait this is the only part that closes the logger, never mind